.prettierrc

```diff
@@ -1,6 +1,7 @@
 {
-    "printWidth": 200,
-    "semi": true,
+    "trailingComma": "all",
     "tabWidth": 4,
-    "singleQuote": true
-}
\ No newline at end of file
+    "singleQuote": true,
+    "semi": true,
+    "printWidth": 90
```

Comment:
This is really too small for me. Do you mind if we keep it at least at 120 or 140?

Reply:
sure, I'll change that to 120